tasks/main.yml

```
@@ -53,6 +53,21 @@
 - name: Run handlers
   ansible.builtin.meta: flush_handlers
 
+- name: Place TLS certificates
+  ansible.builtin.copy:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    owner: bareos
+    group: bareos
+    mode: "0640"
+    backup: "{{ bareos_dir_backup_configurations }}"
+  loop:
+    - bareos_dir_tls_ca_cert
+    - bareos_dir_tls_cert
+    - bareos_dir_tls_cert_key
+  when:
+    - bareos_dir_tls_enable
```

New idea with separate vars in `defaults/main.yml`:

```suggestion
- name: Place TLS certificates
  ansible.builtin.copy:
    src: "{{ item.src }}"
    dest: "{{ item.dest }}"
    owner: bareos
    group: bareos
    mode: "0640"
    backup: "{{ bareos_dir_backup_configurations }}"
  loop:
    - src: bareos_dir_tls_ca_cert_src
      dest: bareos_dir_tls_ca_cert_dest
    - src: bareos_dir_tls_cert_src
      dest: bareos_dir_tls_cert_dest
    - src: bareos_dir_tls_cert_key_src
      dest: bareos_dir_tls_cert_key_dest
  when:
    - bareos_dir_tls_enable
    - item.src != ""
    - item.dest != ""
```